src/server/game/Entities/Unit/Unit.h

```diff
@@ -885,12 +885,26 @@ enum ReactiveType
 
 #define MAX_GAMEOBJECT_SLOT 4
 
-enum PlayerTotemType
+enum PlayerTotemCategory
 {
-    SUMMON_TYPE_TOTEM_FIRE  = 63,
-    SUMMON_TYPE_TOTEM_EARTH = 81,
-    SUMMON_TYPE_TOTEM_WATER = 82,
-    SUMMON_TYPE_TOTEM_AIR   = 83
+    TOTEM_CATEGORY_FIRE     = 3211, // Skyfury Totem
+    TOTEM_CATEGORY_FIRE_2   = 3697, // Liquid Magma Totem
+    TOTEM_CATEGORY_FIRE_3   = 3722, // Voodoo Totem
+
+    TOTEM_CATEGORY_EARTH    = 3400, // Earthbind Totem
+    TOTEM_CATEGORY_EARTH_2  = 3404, // Earthgrab Totem
+    TOTEM_CATEGORY_EARTH_3  = 3737, // Earthen Shield Totem
+    TOTEM_CATEGORY_EARTH_4  = 3787, // Ancestral Protection Totem
+
+    TOTEM_CATEGORY_WATER    = 82,   // Healing Tide Totem
+    TOTEM_CATEGORY_WATER_2  = 3402, // Healing Stream & Cloudburst Totem
+
+    TOTEM_CATEGORY_AIR      = 3399, // Spirit Link Totem
+    TOTEM_CATEGORY_AIR_2    = 3406, // Grounding Totem
+    TOTEM_CATEGORY_AIR_3    = 3407, // Lightning Surge Totem
+    TOTEM_CATEGORY_AIR_4    = 3695, // Winrush Totem
+    TOTEM_CATEGORY_AIR_5    = 3804, // Counterstrike Totem
+    TOTEM_CATEGORY_AIR_6    = 3806  // Windfury Totem
```

Comment:
TOTEM_CATEGORY is a bit misleading, it is a SummonProperty

Reply:
you prefer with enum SummonProperties and SUMMON_PROPERTY_TOTEM_WATER for example?

Reply:
yes